.buildkite/release-automation/wheels.rayci.yml

```diff
@@ -33,13 +47,29 @@ steps:
       - "3.11"
 
   - label: "MacOS x86_64"
+    key: validate-macos-x86_64-wheels
     job_env: MACOS
     instance_type: macos
     commands:
       - ./.buildkite/release-automation/verify-macos-wheels.sh x86_64
 
   - label: "MacOS arm64"
+    key: validate-macos-arm64-wheels
     job_env: MACOS
     instance_type: macos-arm64
     commands:
       - ./.buildkite/release-automation/verify-macos-wheels.sh arm64
+  
+  - block: "Upload wheels to PyPI"
+    depends_on: 
+      - validate-linux-x86-64-wheels
+      - validate-linux-arm64-wheels
+      - validate-macos-x86-64-wheels
+      - validate-macos-arm64-wheels
+
+  - label: "Upload wheels to PyPI"
+    key: upload-wheels-pypi
+    job_env: forge
+    instance_type: small_branch
+    commands:
+      - bazel run //ci/ray_ci/automation:upload_wheels_pypi -- --ray_version="$RAY_VERSION" --commit_hash="$BUILDKITE_COMMIT" --pypi_env=prod
```

Comment:
nit: could you wrap this into multt-line? so that the line is not too long, for better readability?

Reply:
done!